### DISCUSSION TITLE
可否在应用市场中增加一个未安装插件的选项

### DISCUSSION THREAD

**Discussion by Mattoids** [halo-dev/halo#5902] [Ideas]
### 描述一下此特性

现在插件市场只能筛选已安装的插件，但使用过程中，插件商店通常需要看到的是未安装的插件，好方便找到想要安装的插件。已安装的插件可以直接在 插件 界面管理，所以这个功能感觉有点累赘。

**Comment by ruibaby** [comment]
> 插件商店通常需要看到的是未安装的插件

可以考虑加一个是否安装的选项：全部 / 已安装 / 未安装。

> 已安装的插件可以直接在 插件 界面管理，所以这个功能感觉有点累赘

应用市场页面提供的已安装选项只是为了方便勾选之后方便检查当前安装的插件或者主题是否有更新。

当前问题无法在此仓库解决，因为这个改动不涉及到 Halo 仓库，所以我将移动到 Discussion。

**Reply by Mattoids** [reply]
我个人觉得应用市场可以参考 1Panel 的模式，默认情况下只显示未安装插件，需要显示已安装插件直接勾选 “显示本地插件”，这样可以显示全量插件内容。

更新插件目前已经可以在 插件 界面进行更新操作了。所以我个人认为应用市场中增加已安装插件的筛选，从功能上来说有点重复。


以上仅是我个人的一点建议，仅供参考！具体实施还请按照你们的规划来执行即可。

**Reply by ruibaby** [reply]
那可能就和 Halo 本身要耦合了，我们的期望是 Halo 可以完全不依赖应用市场，即不依赖我们提供的服务，独立使用。
